test/catch2/live_restore/live_restore_test_env.cpp

```diff
@@ -23,13 +23,27 @@ live_restore_test_env::live_restore_test_env()
     testutil_recreate_dir(DB_DEST.c_str());
     testutil_recreate_dir(DB_SOURCE.c_str());
 
+    // WiredTiger stores state in the turtle file so we always need to have a valid database in
+    // the source folder. Open and close a connection to initialize the source folder.
+    {
```

Comment:
A side effect of this change is we need a proper WiredTiger database in the source on startup. If we don't I was seeing error messages about the metadata file not existing in the source or destination folders.

I think this is a reasonable constraint, but it's worth keeping in mind when writing tests in future.